### PR TITLE
Minor Corrections

### PR DESCRIPTION
**Pull request by josh-59**
Corrections to foreground example, background example

### CODE REVIEW

**Comment by CoolDeveloper101** [comment]
Hey @josh-59, sorry! I did not see your pull request. Today I will review it and see if it can be merged.

**Comment by CoolDeveloper101** [comment]
Should there be Styles::Normal when there is already a Reset option?

**Comment by josh-59** [comment]
It's your project!  I only thought so because both foreground and
background colors also have 'normal' attributes.

**Comment by CoolDeveloper101** [comment]
Actually, they are quite different, if you visit https://en.wikipedia.org/wiki/ANSI_escape_code and scroll down to SGR parameters, the code for resetting the foreground or background is different and not 0. The argument 0 is used to turn off all properties(including foreground and background) whereas 39 and 49 are used to turn off only the foreground and background Colors respectively.
For example, if I am using a lot of styles but only turn off one of them, then I will use the specific option to turn off that property otherwise if I am using a lot of styles but want to turn them all off then I will use reset.

File: src/styles.rs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 use std::fmt;
 
 pub enum Styles {
+    Normal,
```
Comment (CoolDeveloper101) [inline src/styles.rs]:
```suggestion
	Off
```

File: src/styles.rs [hunk 2]
```diff
@@ -24,6 +25,7 @@ pub enum Styles {
 impl fmt::Display for Styles {
 	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
 	  let code = match self {
+        Styles::Normal => 0,
```
Comment (CoolDeveloper101) [inline src/styles.rs]:
```suggestion
        Styles::Off=> 0,
```

**Pull request merged**